themes/digital.gov/src/scss/new/shortcodes/_highlight.scss

```diff
@@ -1,7 +1,12 @@
 @use "uswds-core" as *;
 
 .highlight {
-  @include u-padding-y("1px");
-  @include u-padding-x("05");
-  @include u-bg("yellow-10v");
+  @include u-padding-y(1);
+  @include u-padding-x(2);
+  @include u-bg("blue-warm-5");
+
+  code {
+    background-color: #fff;
+    border-style: none;
+  }
```

Comment:
Shouldn't we set this on **all** code elements (including inline)?

I also see a border on the code block. Why are we removing the border style?

**Inline code**
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/7ab9f22c-7980-4420-bd00-93d31a7cf89d)
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/7bd26560-3c6a-4e74-b84a-542edb71262e)


**Block code**
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/5b3aa262-4629-4594-9a28-5dd923d4799c)

Reply:
Thats a good question... so before the removal of the border sytle there were these small grey boxes there were getting applied and I targeted the code blocks within the highlight class.  this way the other code blocks would be left alone. 

here is a screen shot of what it looked like before... 


Screenshot 2024-04-23 at 12.53.14 PM.png

Reply:
That's coming from custom overrides in `_content.scss`.

We'll need to add something like this:
```diff
-  // Code
+ // Inline code.
+ // Example: `/resources/guide-create-mobile-friendly-websites`
-  code {
+  p > code {
```